Formula/pyqt.rb

```diff
@@ -13,32 +14,89 @@ class Pyqt < Formula
     sha256 mojave:        "cdb9e6b67c56f66953beae5c5882bd718dfdf17c7a77036d7aadf3c86a4f676d"
   end
 
-  depends_on "pyqt-builder" => :build
-  depends_on "sip" => :build
-
   depends_on "python@3.9"
   depends_on "qt"
 
+  # build dependencies
+  resource "packaging" do
+    url "https://files.pythonhosted.org/packages/86/3c/bcd09ec5df7123abcf695009221a52f90438d877a2f1499453c6938f5728/packaging-20.9.tar.gz"
+    sha256 "5b327ac1320dc863dca72f4514ecc086f31186744b84a230374cc1fd776feae5"
+  end
+
+  resource "pyparsing" do
+    url "https://files.pythonhosted.org/packages/c1/47/dfc9c342c9842bbe0036c7f763d2d6686bcf5eb1808ba3e170afdb282210/pyparsing-2.4.7.tar.gz"
+    sha256 "c203ec8783bf771a155b207279b9bccb8dea02d8f0c9e5f8ead507bc3246ecc1"
+  end
+
+  resource "pyqt-builder" do
+    url "https://files.pythonhosted.org/packages/12/29/af52add4755b7dbce928fe3df1b86a48d5c8bcd06e3333bf8f69ad19c1a5/PyQt-builder-1.10.0.tar.gz"
+    sha256 "86bd19fde83d92beaefacdeac1e26c6e1918c300ff78d7ec2a19973bf2cf21b5"
+  end
+
+  resource "sip" do
```

Comment:
Try to reduce the dependencies.
[`sip` 6.x](https://github.com/Homebrew/homebrew-core/commit/dd5327d63e2fd45a449eeae7df983724e54b22e6) doesn't comply with homebrew python formula policies, though `sip` is an old formula since Homebrew started providing `pyqt`.
```
Homebrew draws a distinction between Python applications and Python libraries.
```
`pyqt` need `sip` applications and libraries to build.

Reply:
But it doesn't reduce the dependencies. You just add it to the formula. That means more downloads, and more time spent building software. `sip` is already a formula. We should make use of it.